functions/controllers/application/index.js

```diff
@@ -111,18 +111,19 @@ application.get("/analytics", jwtCheck, hasReadAnalytics, async (req, res) => {
   try {
     const analyticsSnapshot = await queryDocument("analytics", "applicant-analytics");
     if (!analyticsSnapshot.exists) {
-      throw "No Doc";
+      throw new Error("No Document");
     }
+
     res.status(201).send({
       message: {
         applicantCount: analyticsSnapshot.get("applicantCount"),
         firstTime: analyticsSnapshot.get("firstTimeCount"),
         ucscApplicants: analyticsSnapshot.get("ucscStudentCount"),
       },
     });
-  } catch (err) {
-    if (err === "No Doc") {
-      res.status(404).send({ status: 404, message: "No Document" });
+  } catch (error) {
+    if (error.message === "No Document") {
+      res.status(200).send({ status: 404, message: "No Document" });
```

Comment:
Make sure to have .`status()` and `res.body.status` equal in the response code here as well

Reply:
I think in this situation 200 or 404 is valid for a response code since no document is an error so I'm indifferent for this endpoint.